src/org/daisy/braille/utils/pef/TextHandler.java

```diff
@@ -375,7 +377,11 @@ public void parse() throws IOException {
         if (!"".equals(language)) {
             pw.println("            <dc:language>" + language + "</dc:language>");
         }
-        pw.println("            <dc:date>" + DATE_FORMAT.format(date) + "</dc:date>");
+
+        synchronized (DATE_FORMAT) {
```

Comment:
Blocks should be synchronized on `private final` fields.

Reply:
Changed DATE_FORMAT to a private final variable.